lenskit-predict/src/main/java/org/grouplens/lenskit/predict/KnownRatingRatingPredictor.java

```diff
@@ -0,0 +1,43 @@
+package org.grouplens.lenskit.predict;
+
+import it.unimi.dsi.fastutil.longs.LongSortedSet;
+import org.grouplens.lenskit.basic.AbstractRatingPredictor;
+import org.grouplens.lenskit.data.dao.UserEventDAO;
+import org.grouplens.lenskit.data.event.Rating;
+import org.grouplens.lenskit.vectors.MutableSparseVector;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+import java.util.List;
+
+/**
+ * Created by Tajinder on 1/6/2015.
+ * 'KnownRatingRatingPredictor' predicts the known ratings of each item and the user given in the predictions
```

Comment:
Let's rewrite this comment. A couple of general principals:

The first sentence will be a brief summary. It should *not* include the class name, since the class name will always be displayed with it.

So something like this:

```
/**
 * Use a user's existing ratings as predictions.  This rating predictor 
 * returns the user's rating as a prediction for all items the user has rated, 
 * and returns no prediction for items the user has rated. It is useful in
 * conjunction with a {@link FallbackItemScorer} to return a user's rating
 * when it is known.
 *
 * @see RatingPredictorItemScorer
 * @since 2.2
 */